packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -225,6 +226,11 @@ class VideoEvent {
   /// Only used if [eventType] is [VideoEventType.initialized].
   final Size? size;
 
+  /// Degrees to rotate the video so it is displayed correctly.
```

Comment:
Nit, based on something that caused confusion in a different rotation-related PR recently (for camera): please specify a direction—I assume clockwise—so there's no possible ambiguity.